diff --git a/node_modules/mixpanel-react-native/android/build.gradle b/node_modules/mixpanel-react-native/android/build.gradle
index 8a404a5..0848055 100644
--- a/node_modules/mixpanel-react-native/android/build.gradle
+++ b/node_modules/mixpanel-react-native/android/build.gradle
@@ -11,11 +11,11 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 33
     buildToolsVersion "28.0.3"
     defaultConfig {
         minSdkVersion 21
-        targetSdkVersion 26
+        targetSdkVersion 33
         versionCode 1
         versionName "1.0"
     }
